pages/cloudflare/get-started.mdx

```diff
@@ -118,70 +143,42 @@ Add the following to the scripts field of your `package.json` file:
 - `npm run preview:worker`: Runs `build:worker` and then `dev:worker`, allowing you to quickly preview your app running locally in the Workers runtime, via a single command.
 - `npm run deploy`: Builds your app, and then deploys it to Cloudflare
 
-### 6. Add caching with Workers KV
+##### 7. Add caching with Workers KV
 
 See the [Caching docs](/cloudflare/caching) for information on enabling Next.js caching in your OpenNext project.
 
-### 7. Remove `@cloudflare/next-on-pages` (if necessary)
-
-If your Next.js app currently uses `@cloudflare/next-on-pages`, you'll want to remove it, and make a few changes.
+##### 8. Remove any `export const runtime = "edge";` if present
 
-#### Remove `export const runtime = "edge";`
+Before deploying your app, remove the `export const runtime = "edge";` line from any of your source files.
 
-Before deploying your app, remove the `export const runtime = "edge";` line from your `next.config.js` file.
 The edge runtime is not supported yet with `@opennextjs/cloudflare`.
 
-#### Add `.open-next` to `.gitignore`
+##### 9. Add `.open-next` to `.gitignore`
 
 You should add `.open-next` to your `.gitignore` file to prevent the build output from being committed to your repository.
 
-#### Uninstall `@cloudflare/next-on-pages`
```

Comment:
I'm trying to clean things up a bit here

We have both a `Remove '@cloudflare/next-on-pages' (if necessary)` and a `Uninstall '@cloudflare/next-on-pages'` section, this seems pretty confusing to me, also the hierarchy is a bit out of whack I think (for example, `Add '.open-next' to '.gitignore'` is under `Remove '@cloudflare/next-on-pages' (if necessary)`, that doesn't seem correct to me)